### PR TITLE
Add jobs

### LINKED ISSUE #10: Create data rendering for readme & PM review

### LINKED ISSUE #12: create database
---
Name: create database
About: PostgresQL

---

### User Story
chore

### LINKED ISSUE #13: /jobs endpoint
---
Name: Create '/jobs' endpoint for FE consumption
About: Requirements found here :https://docs.google.com/spreadsheets/d/1mxnKK4iQ-wrJeZTF3zWcYskbkqlm4G4S5A7gXrfFMy8/edit#gid=483454443

---

### User Story
The /jobs endpoint will be an index view of the short descriptions for all jobs in the db. see attached for other fields to be included. 
### Requirements
see attached

### LINKED ISSUE #14: /job/:id endpoint
---
Name: /job/:id endpoint
About:  Requirements found here: https://docs.google.com/spreadsheets/d/1mxnKK4iQ-wrJeZTF3zWcYskbkqlm4G4S5A7gXrfFMy8/edit#gid=483454443

---

### User Story
The /job/:id endpoint will be a show view of the long descriptions for all jobs in the db. see attached for other fields to be included.

### PR DESCRIPTION
**Pull request by janegreene**
# Description
Stood up Flask App and added end points for all jobs and job show page. 
See readme for set-up instructions

# Closes issue(s)
#10, #12, #13, #14 
# How to test / reproduce

# Screenshots
Screen Shot 2020-10-24 at 3 41 18 PM
Screen Shot 2020-10-24 at 3 41 57 PM

# Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist
- [ ] I have tested this code
- [x] I have updated the Readme

# Other comments
Tests to be completed still.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+{
+    "jobs": [
+        {
+            "avg_salary": "47,500",
+            "education": "Bachelor's degree in graphic design",
+            "icon_url": "https://www.flaticon.com/svg/static/icons/svg/2192/2192721.svg",
+            "id": 1,
+            "short_description": "Graphic designers develop graphics and layouts for product illustrations, company logos, websites and more.",
+            "title": "Graphic Designer"
+        },...
```
Comment (janegreene) [inline README.md:40]: This continues for 15 jobs at the moment.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Flask
```
Comment (dtsehl) [inline requirements.txt:1]: It's very cool that you can add in the requirements in a .txt file and then just `flask install requirements.txt` to install them all.

File: test.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import requests
```
Comment (dtsehl) [inline test.py]: Whoops! Maybe we don't need this file in here. It's based on the videos test file.

**Review by dtsehl — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,61 @@
...
+        "id": 1,
+        "image_url": "https://www.freepik.com/blog/app/uploads/2018/07/PIN-BLOG-1270x720-What-is-the-difference-between-an-illustrator-and-a-graphic-designer-1.jpg",
+        "long_description": "Graphic designers are visual communicators, who create visual concepts by hand or by using computer software. They communicate ideas to inspire, inform, or captivate consumers through both physical and virtual art forms that include images, words, or graphics. The end goal of graphic designers is to make the organizations that hire them more well known by their designs. By using a variety of media they communicate a particular idea to be used in advertising and promotions. These media include fonts, shapes, colors, images, print design, photography, animation, logos, and billboards. Graphic designers often work on projects with artists, multimedia animators and other creative professionals.",
+        "max_salary": "60,000",
+        "min_salary": "35,000",
+        "title": "Graphic Designer",
+        "video_url": "https://www.youtube.com/watch?v=kqwgs7vBMkU"
+    }
+}
+```
```
Comment (Strompy) [inline README.md:61]: This is great! I usually wait too long before starting the readme

File: migrations/env.py [hunk 5]
```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
+
```
Comment (Strompy) [inline migrations/env.py:2]: Was this file generated for us?

File: main.py [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+        long_description="Pharmacists are healthcare practitioners who check and distribute drugs to doctors for medication that had been prescribed to patients. They advise patients and health care providers on the selection, dosages, interactions, and side effects of medications. They might work in a  hospital or retail store."
+    )
+    jobs = [job1, job2, job3, job4, job5, job6, job7, job8, job9, job10, job11, job12, job13, job14, job15]
+    for job in jobs:
+        db.session.add(job)
+    db.session.commit()
+    print('Database seeded!')
+
+@app.route('/jobs')
+def handle_jobs():
```
Comment (Strompy) [inline main.py:232]: One of the next steps we can do this week is refactor some of these into their own files

**Review by Strompy — APPROVED** [review]
This is coming together nicely!

**Pull request merged**